diffengine/__init__.py

```diff
@@ -431,14 +436,29 @@ def get_initial_config():
         consumer_secret = input("What is the consumer secret? ")
 
         token = request_pin_to_user_and_get_token(consumer_key, consumer_secret)
-
+        config["publishing_handlers"].append("Twitter")
```

Comment:
This would append a new twitter handler per every feed you have in the array. This should be outside the `while` clause